pkg/minishift/constants/constants.go

```diff
@@ -53,7 +53,10 @@ func ProfilePrivateKeyPath() string {
 	return filepath.Join(constants.Minipath, "certs", "id_rsa")
 }
 
-func GetOpenshiftImageName(openshiftVersion string) string {
+func GetOpenshiftImageToFetchOC(openshiftVersion string, isGreaterOrEqualToBaseVersion bool) string {
+	if isGreaterOrEqualToBaseVersion {
+		return fmt.Sprintf("openshift/origin-control-plane:%s", openshiftVersion)
```

Comment:
@praveenkumar I thought we are going to use `origin-cli image`?

Reply:
@LalatenduMohanty https://github.com/openshift/origin/issues/20892 but `origin-control-plane` is always updated and used for initial openshift cluster start.